### ISSUE TITLE
Stream features to offline storage

### ISSUE THREAD

**Issue by hangfei** [linkedin/feathr#530]
We now support stream features to online table(Redis). 

It's a popular ask by users.

The main code is added by this https://github.com/linkedin/feathr/pull/137.
Existing user guide: https://github.com/linkedin/feathr/blob/f3d6ba57425d550f19730403115c0523f6c3ffe6/docs/how-to-guides/streaming-source-ingestion.md.

**Comment by hangfei** [comment]
@ahlag If you are interested, feel free to assign this to yourself.

**Comment by ahlag** [comment]
@hangfei 
Yeah sure! By the way, I can't assign myself. I think it's because I don't have the permission.

**Comment by hangfei** [comment]
#289 provide an API to "query" offline data.. This provides an API to write streaming data to offline.

**Comment by gkorland** [comment]
Did you consider having [Redis Streams](https://redis.io/docs/data-types/streams/) as a lightweight alternative Stream to Kafka.
Notice Redis already has a nice integration with [Spark Streaming ](https://github.com/RedisLabs/spark-redis/blob/master/doc/streaming.md)
